mypyc/test/test_exceptions.py

```diff
@@ -32,7 +32,9 @@ def run_case(self, testcase: DataDrivenTestCase) -> None:
         """Perform a runtime checking transformation test case."""
         with use_custom_builtins(os.path.join(self.data_prefix, ICODE_GEN_BUILTINS), testcase):
             expected_output = remove_comment_lines(testcase.output)
-
+            # replace native_int with platform specific ints
+            int_format_str = 'int32' if IS_32_BIT_PLATFORM else 'int64'
+            expected_output = [s.replace('native_int', int_format_str) for s in expected_output]
```

Comment:
Since we do this same dance in multiple test files, maybe you can move this to a helper function? This way if we modify this, we only have to change it in one place.